inc/admin/attachments/namespace.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * @author Brad Payne, Alex Paredes
+ * @license  GPLv3 (or any later version)
+ */
+
+namespace Pressbooks\Admin\Attachments;
+
+use Pressbooks\Licensing;
+use Pressbooks\Utility;
+
+/**
+ * Hooks into 'attachment_fields_to_edit' filter to add custom attachment
+ * metadata. Currently, on the display side of capturing this metadata
+ * unlike images, there is no easy way to determine an audio or video post id
+ * from the markup that's generated: (wp-image-33), therefore this can only
+ * apply to images, for now
+ *
+ * @since 5.5.0
+ *
+ * @param $form_fields
+ * @param $post
+ *
+ * @return mixed
+ */
+function add_metadata_attachment( $form_fields, $post ) {
+
+	if ( substr( $post->post_mime_type, 0, 5 ) === 'image' ) {
+
+		$author      = get_post_meta( $post->ID, 'pb_media_attribution_author', true );
+		$author_url  = get_post_meta( $post->ID, 'pb_media_attribution_author_url', true );
+		$source      = get_post_meta( $post->ID, 'pb_media_attribution_title_url', true );
+		$license     = get_post_meta( $post->ID, 'pb_media_attribution_license', true );
+		$figure      = get_post_meta( $post->ID, 'pb_media_attribution_figure', true );
+		$adapted     = get_post_meta( $post->ID, 'pb_media_attribution_adapted', true );
+		$adapted_url = get_post_meta( $post->ID, 'pb_media_attribution_adapted_url', true );
+
+		$form_fields['pb_attribution'] = [
+			'value' => '',
+			'label' => __( 'ATTRIBUTIONS', 'pressbooks' ),
+			'input' => 'html',
+			'html'  => '<span></span>',
+		];
+
+		$form_fields['pb_media_attribution_figure'] = [
```

Comment:
My preference would be to remove the figure number field and consider it as a separate issue; dynamic numbering of figures would probably be a better user experience so if this isn't required for the functionality contained in this PR it would be best to omit it (see also: https://github.com/pressbooks/ideas/issues/114).